app/graphql/mutations/data_exports/invoices/create.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Mutations
+  module DataExports
+    module Invoices
+      class Create < BaseMutation
+        include AuthenticableApiUser
+        include RequiredOrganization
+
+        REQUIRED_PERMISSION = 'invoices:create'
```

Comment:
still investigating how the permissions work. The feature should be available for all roles...

Reply:
I'd recommend creating `invoices:export`. You can add to the [permissions definitions.yml](https://github.com/getlago/lago-api/blob/main/app/config/permissions/definition.yml#L40-L45) file.

When set to true here, it's available to all roles.

```
invoices:
  - ...
  - export: true
```

Reply:
thanks